From: David Paleino <dapal@debian.org>
Subject: fix typo spot by lintian
Forwarded: no

---
 src/rasterlite_pyramid.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/src/rasterlite_pyramid.c
+++ b/src/rasterlite_pyramid.c
@@ -1029,7 +1029,7 @@ build_pyramid_level (sqlite3 * handle, c
 	  goto error2;
       }
 
-    printf ("Pyramid Level %d succesfully created\n", level);
+    printf ("Pyramid Level %d successfully created\n", level);
     printf ("ThumbnailTiles:     %d rows in table \"%s_rasters\"\n", raster_ok,
 	    table);
     printf ("                    %d rows in table \"%s_metadata\"\n",
